includes/admin/post-types/meta-boxes/class-ph-meta-box-enquiry-details.php

```diff
@@ -326,12 +326,12 @@ function viewing_update_selected_properties()
                         $right_padding = '105px';
                     }
 
-                    echo '<a href="' . get_edit_post_link($enquiry_contact_id, '') . '" class="button" style="position:absolute; top:0; right:' . $right_padding . ';">' . __( 'View ' . $enquiry_contact_type, 'propertyhive' ) . '</a>';
+                    echo '<a href="' . get_edit_post_link($enquiry_contact_id, '') . '" class="button" style="position:absolute; top:0; right:' . $right_padding . ';">' . __( 'View ', 'propertyhive' ) . $enquiry_contact_type . '</a>';
```

Comment:
I'm not sure how it was before was 'wrong'. There are a finite number of contact types and this allows you to translate these also. Otherwise the $enquiry_content_type would remain untranslated if translation is the goal.

Reply:
Hello @propertyhive 

Can the value of $enquiry_contact_type be 'Applicant' or 'Contact'? Is that so?

The string 'View Contact' is present among the translatable strings, but the string 'View Applicant' is not.

In any case, I think that a variable inside the __() function, even if the translation is displayed, can create confusion and problems..

What do you think?

Thanks.